attest/verifier/src/dcap.rs

```diff
@@ -45,8 +45,8 @@ impl DcapVerifier {
         I: IntoIterator<Item = ID>,
         ID: Into<TrustedIdentity>,
     {
-        let trust_anchor = TrustAnchor::try_from_pem(IAS_SIGNING_ROOT_CERT_PEM)
-            .expect("Failed to parse root cert");
+        let trust_anchor =
+            TrustAnchor::try_from_pem(DCAP_ROOT_ANCHOR).expect("Failed to parse root cert");
```

Comment:
:facepalm: Just noticed that we were using the IAS root CA, which works in sim builds but would fail once we get this running on HW.